recipes/poco/all/conanfile.py

```diff
@@ -230,7 +230,8 @@ def package_info(self):
             if comp.option is None or self.options.get_safe(comp.option):
                 conan_component = "poco_{}".format(compname.lower())
                 requires = ["poco_{}".format(dependency.lower()) for dependency in comp.dependencies] + comp.external_dependencies
-                self.cpp_info.components[conan_component].set_property("cmake_target_name", compname)
+                self.cpp_info.components[conan_component].set_property("cmake_target_name", "Poco::{}".format(compname))
+                self.cpp_info.components[conan_component].set_property("cmake_file_name", compname)
```

Comment:
What is the meaning of `cmake_file_name` for components?

Reply:
I think that's a mistake, it won't have effect but it should not be set for components.